tests/test_homo_sapiens.py

```diff
@@ -53,6 +55,40 @@ def test_chromosome_lengths(self):
         self.assertEqual(genome.chromosomes["chrX"].length, 156040895)
         self.assertEqual(genome.chromosomes["chrY"].length, 57227415)
 
+    def test_chromosome_recombination_maps(self):
+
+        with self.assertWarns(Warning):
```

Comment:
This assertWarns is pretty scattergun, and it's not at all obvious why it should raise a warning. You need to make it locally obvious what code is doing, a reader shouldn't need to keep the whole codebase in their head to understand a local function.

Really you want  to split this into two unit tests, one where we're testing chromosomes where we know we have maps and another where we know we don't.

Reply:
Agreed, I think here I wanted to make sure that every `chrom in homo_sapiens.genome.chromosomes:` could call `chrom.recombination_map()`  and get an instance of `msprime recombination_map` with nothing *more* than a warning. That is the desired behavior?

I'll break the tests up to make it a little more concise.

Reply:
OK, cool. Ping me when it's ready to look at again.

Reply:
@jeromekelleher ping

Reply:
Great! OK, looks good to go for me. Thanks for the updates.